lib/models/quellenreiter_app_state.dart

```diff
@@ -435,6 +429,8 @@ class QuellenreiterAppState extends ChangeNotifier {
     // update player and opponent here, to be up to date with played statements
```

Comment:
the risk here is not anymore overwriting anything. Instead, if in the exact moment the other player adds some playedStatements to the DB, it can happen, that som of those will get selected for this game as well. (not too bad and very unlikely)